include/openthread/trel.h

```diff
@@ -149,6 +162,26 @@ void otTrelSetFilterEnabled(otInstance *aInstance, bool aEnable);
  */
 bool otTrelIsFilterEnabled(otInstance *aInstance);
 
+/**
+ * Returns TREL counters.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ *
+ * @returns  A pointer to the TREL counters.
+ *
+ */
+const otTrelCounters *otTrelGetCounters(otInstance *aInstance);
+
+/**
+ * Returns the number of TREL peers.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ *
+ * @returns  The number of TREL peers.
+ *
+ */
+uint16_t otTrelGetPeersNum(otInstance *aInstance);
```

Comment:
Move after `otTrelGetNextPeer()` definition (so related APIs are close to each other).

Reply:
Done